cakephpsphinx/themes/cakephp/static/opensearchdescription-book-4-x.xml

```diff
@@ -1,12 +1,12 @@
 ﻿<?xml version="1.0" encoding="UTF-8"?>
 <OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/">
-  <ShortName>CakePHP Book 3.x</ShortName>
+  <ShortName>CakePHP Book 4.x</ShortName>
```

Comment:
This should also be dynamic based on the version, but that is something I can take care of.

Reply:
Shouldn't this not be in the sphinxtheme anyway since it's specific to the docs? I was doing this as a bugfix rather than improvement for now.

Reply:
We use this theme for a variety of plugin docs as well. Fixing this will allow those plugins to have a search provider as well.